guide/case-studies/blixt-wallet.md

```diff
@@ -0,0 +1,173 @@
+
+The current user base of Blixt tends to be more advanced and Blixt would like to move towards catering to beginners. 
+
+What does Blixt do?
+
+- Onboard someone quickly over lightning so they can have a channel already provisioned to receive funds
+- Automatically open channels anytime the user has on-chain funds - [Auto pilot](https://bitcoin.design/guide/how-it-works/lightning-services/#on-chain-funding)
+- Automatic channel creation (using [Dunder LSP](https://github.com/hsjoberg/dunder-lsp)) during initial deposit of funds
+- Support for custom LSP
+
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-users.jpg"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-users@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-users-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-users.jpg"
+   alt-text = "Characters representing Blixt's users"
+   caption = "Illustrations: [OpenPeeps](https://www.openpeeps.com/) by Pablo Stanley"
+   width = 800
+   height = 450
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process.jpg"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process.jpg"
+   alt-text = "Diagram of stages in the Blixt UX process"
+   width = 800
+   height = 929
+%}
+
+Since Blixt is an open source wallet, we collected user research simply by going on their Discord channel and collecting user comments and questions.
+
+## We discovered that Blixt users were struggling to open lightning channels
+
+We created a lot of different user flows to find a solution for the best method to open a channel for a user new to lightning, and eventually settled on this one. We decided on this user flow as it educates users first and then takes them to setting up the channel itself. 
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-user-flow.png"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-user-flow@2x.png"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-user-flow-mobile.png"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-user-flow-mobile@2x.png"
+   alt-text = "Diagram of the Blixt user flow, from initial onboarding through setting up channels, backing up the wallet, scanning a QR, and sending first payment."
+   width = 1600
+   height = 644
+   layout = "full-width"
+%}
+
+## One of the biggest challenges we faced was
```

Comment:
```suggestion
## One of the biggest challenges
```
It's weird how the title just ends. I understand it's meant to lead into the paragraph. It might work with a '...' at the end of the title, but without it it's weird.

Reply:
Fixed, but we tried using a colon. What do you think?

(We can play with an ellipsis some more, but it causes unexpected linebreaks)